App/TurboController.swift

```diff
@@ -325,6 +325,11 @@ extension TurboController: SessionDelegate {
             self.loadButtons(authenticated == "true")
         })
     }
+
+    // added to turbo-ios in 7.0.0-rc.6
+    func sessionWebViewProcessDidTerminate(_ session: Session) {
+        // TODO
+    }
```

Comment:
this is the meaningful piece per https://github.com/hotwired/turbo-ios/compare/7.0.0-rc.5...7.0.0-rc.6#diff-b8eedf003187f841896b81cea7ce824a547fb8f3cc3ad74717f09e3bdbadebb9R14

Reply:
@dalezak This is useful, I had to do the same.

```swift

    func startApp() {
        // app setup
        self.navigationController?.setNavigationBarHidden(true, animated: false)
        self.navigationController?.view.backgroundColor = UIColor(hexRGB: "#121212")
        loadNavBar()
        loadTabBar()
        loadTabs()
        loadHome()     
    }

override func viewDidLoad() {
        super.viewDidLoad()
        startApp()
    }

    func sessionWebViewProcessDidTerminate(_ session: Session) {
        self.navigationController?.popViewController(animated: false)
        self.navigationController?.viewControllers = []
        self.tabBar = makeTabBar()
        self.session = makeSession()
        self.modalSession = makeSession()
        self.settings = loadSettings()
        self.processPool = WKProcessPool()
        startApp()
    }
```